hc/front/views.py

```diff
@@ -211,11 +211,12 @@ def channels(request):
                     check = Check.objects.get(code=code)
                 except Check.DoesNotExist:
                     return HttpResponseBadRequest()
-                if check.user != request.user:
+                if check.user_id != request.user.id:
                     return HttpResponseForbidden()
                 new_checks.append(check)
 
-        channel.checks = new_checks
```

Comment:
does django really do what u think it does when u do this kind of direct assignment?